cmds/exp/cpu/doc.go

```diff
@@ -126,4 +147,74 @@
 //    rminnich@xcpu:~/gopath/src/github.com/u-root/u-root/xcmds/cpu$ ls -l ~/xyz
 //    -rw-r--r-- 1 rminnich rminnich 577 May 17 17:06 /home/rminnich/xyz
 //    rminnich@xcpu:~/gopath/src/github.com/u-root/u-root/xcmds/cpu$
+//
+// A note on sizes of things. We can get an image down to 3 MiB if the only
+// binary is cpu. See github.com:linuxboot/mainboars/aeeon/up for an example.
```

Comment:
mainboards

Reply:
Fixed, though now I want to work on mainboars. Whatever animal that is.